src/main/config/config.c

```diff
@@ -660,11 +666,6 @@ static void resetConf(void)
 
 #if defined(ALIENFLIGHT) 
     featureClear(FEATURE_ONESHOT125);
-#ifdef ALIENFLIGHTF1
-    masterConfig.serialConfig.portConfigs[1].functionMask = FUNCTION_RX_SERIAL;
-#else
-    masterConfig.serialConfig.portConfigs[2].functionMask = FUNCTION_RX_SERIAL;
```

Comment:
@MJ666 , most of the target specific stuff has moved now. There is also a "catch all" item in `config.c`
```
#if defined(TARGET_CONFIG)
    targetConfiguration(&masterConfig);
#endif
```
So for unusual cases what you can do is define `TARGET_CONFIG` in `target.h` and then, in the target directory create a `config.c` file which includes the function `targetConfiguration(master_t *masterConfig)` - this would be a good place to put, for example, the custom mixer.

The other thing is that parameter groups are coming at some point, and they should help with eliminating some of the cases.